tests/integration/sriov_test.py

```diff
@@ -268,3 +268,34 @@ def test_wait_sriov_vf_been_deleted_when_total_vfs_decrease():
             Interface.STATE
         ] = InterfaceState.ABSENT
         libnmstate.apply(desired_state)
+
+
+@pytest.mark.skipif(
+    not os.environ.get("TEST_REAL_NIC"),
+    reason="Need to define TEST_REAL_NIC for SR-IOV test",
+)
+def test_create_new_vfs_does_not_generate_a_profile():
+    pf_name = _test_nic_name()
+    desired_state = {
+        Interface.KEY: [
+            {
+                Interface.NAME: pf_name,
+                Ethernet.CONFIG_SUBTREE: {
+                    Ethernet.SRIOV_SUBTREE: {Ethernet.SRIOV.TOTAL_VFS: 2}
+                },
+            }
+        ]
+    }
+    try:
+        libnmstate.apply(desired_state)
+        assert not os.path.isfile(
+            f"/etc/sysconfig/network-scripts/ifcfg-{pf_name}v0"
```

Comment:
ifcfg will not be used in Fedora 34+ any more, please use `nmcli c` to check the existence of NM profile.

This test case is NM specific, please move to `test/integration/nm` folder.

Reply:
Done, I've created `test/integration/nm/sriov_test.py`. Thanks